blender/arm/logicnode/depreciated/LN_get_mouse_lock.py

```diff
@@ -0,0 +1,26 @@
+import bpy
+from bpy.props import *
+from bpy.types import Node, NodeSocket
+from arm.logicnode.arm_nodes import *
+
+class GetMouseLockNode(ArmLogicTreeNode):
+    bl_idname = 'LNGetMouseLockNode'
+    bl_label = 'Get Mouse Lock (Depreciated)'
```

Comment:
Just a small reminder that those typos still need to be fixed :)

Reply:
oh, sorry. It's fixed now.